packages/@okta/i18n/src/properties/login_ok_PL.properties

```diff
@@ -804,3 +804,4 @@ authfactor.challenge.soft_token.invalid_passcode = 》Ýöûŕ çöðé ðöéš
 authfactor.challenge.soft_token.used_passcode = 》Éåçĥ çöðé çåñ öñļý ƀé ûšéð öñçé· Þļéåšé ŵåîţ ƒöŕ å ñéŵ çöðé åñð ţŕý åĝåîñ· 홝한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
 oie.tooManyRequests = 》Ţöö ɱåñý ŕéǫûéšţš 한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
 E0000010 = 》Šéŕṽéŕ îš ûñåƀļé ţö ŕéšþöñð åţ ţĥé ɱöɱéñţ· ヸ€홝한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
+idx.error.code.user_not_assigned = 》Ýöû åŕé ñöţ åļļöŵéð ţö åççéšš ţĥîš ţĥîš åþþ· Ţö ŕéǫûéšţ åççéšš، çöñţåçţ åñ åðɱîñ· 한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
```

Comment:
This string was manually created, not generated with pseudo-loc

Reply:
@denysoblohin-okta curious why we had to generate this manually ?  There was a bug in the script that I fixed recently https://github.com/okta/okta-signin-widget/pull/2000

Reply:
I can't install `pseudo-loc` with `yarn install`. 
I get error `https://artifacts.aue1d.saasure.com:443/artifactory/api/npm/npm-okta-all/@okta/app-localizer/-/@okta/app-localizer-1.2.3-105-g97aab49.tgz: getaddrinfo ENOTFOUND artifacts.aue1d.saasure.com`

Reply:
Yes, this requires VPN as it is an internal package

Reply:
@denysoblohin-okta  what you have generated manually seems fine. If you need help generating this using the script let me know and I can checkout your branch and run the script.